### PR TITLE
dockerfile-mode 1.2 (new formula)

### PR DESCRIPTION
**Pull request by rwhogg**
This adds a package containing a major mode for editing Dockerfiles.

### CODE REVIEW

File: Formula/dockerfile-mode.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class DockerfileMode < EmacsFormula
+  desc "Emacs mode for handling Dockerfiles "
+  homepage "https://github.com/spotify/dockerfile-mode"
+  url "https://github.com/spotify/dockerfile-mode/archive/v1.2.tar.gz"
+  sha256 "49618f06ed6d7a4d64251e00540df13870aeee9a8f55acd4def0482ada78156e"
+  head "git@github.com:spotify/dockerfile-mode.git"
```
Comment (dunn) [inline Formula/dockerfile-mode.rb]: Can you change this to use the `https://` URL?

Comment (rwhogg) [inline Formula/dockerfile-mode.rb]: Done and force-pushed.

**Comment by dunn** [comment]
Cool, thanks!

Looks like the Linux CI isn't going to be very useful until Linuxbrew starts getting merged regularly again: https://travis-ci.org/Homebrew/homebrew-emacs/jobs/98634770

**Comment by rwhogg** [comment]
Yeah, I tried my best to find a way to work around that, but unfortunately trying to cherry-pick the commit that gets rid of all those errors was a PITA.

**Comment by dunn** [comment]
Not your fault! Shaun will be back in the new year, so it should be resolved soon.

**Pull request closed without merging**